Extensions/alternate_audio_player.js

```diff
@@ -0,0 +1,60 @@
+//* TITLE Alternate Audio Player **//
+//* VERSION 0.1.0 **//
+//* DESCRIPTION	Adds an alternative (working) audio player to audio posts. **//
+//* DETAILS This extension adds a native HTML5 audio player to every audio post, which should always work as well as enabling looping, downloading, and better seeking. **//
+//* DEVELOPER akunohomu **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.alternate_audio_player = new Object({
+
+	running: false,
+
+	preferences: {
+		"default_volume": {
+			text: "Default volume",
+			default: 1.0,
+			value: 1.0,
+			type: "combo",
+			values: [
+				"100%", 1.0,
+				"75%", 0.75,
+				"50%", 0.50,
+				"25%", 0.25
+			],
+			desktop_only: true
```

Comment:
Confusion. But this entire extension is unnecessary on the mobile site - Tumblr already uses a native player there.